chart/templates/service.yaml

```diff
@@ -4,6 +4,10 @@ kind: Service
 metadata:
   name: jspolicy
   namespace: {{ .Release.Namespace }}
+ {{- if (.Values.service).annotations }}
```

Comment:
Why is the brackets between (.Values.service) needed? Also for styling reasons it would be good to add one more whitespace here to align with the instructions below